### ISSUE TITLE
Ea app version when?

### ISSUE THREAD

**Issue by justcallmedots** [marcos4503/ts3-dream-launcher#9]
I paid for ea play so i wanted to play sims 3. my question is will we get EA app's version compatible by the end of this year? also thanks very much for the attempt to make this possible.

**Comment by marcos4503** [comment]
Unfortunately I can only support The Sims 3 of Steam. As for the EA App/Origin or Pirate, I have not tested it and I cannot say when this support will come, as I will not be able to offer support for these.

Even if I decided to port Dream Launcher to support the EA/Origin or Pirate version, I wouldn't have time to maintain or even update the program if anything new was released in The Sims 3, such as hardware changes that require new patches, for example.

I would also need to spend more time porting any new changes I made, or new features I added, and so on... So, taking this into consideration, if I decided to port it to EA/Origin or Pirate, to simply abandon the program in disrepair, that would go against my philosophy. 😕

That's why I prefer to only keep what I can take care of and give attention to 100% calmly. It's also important to take into consideration that Dream Launcher is my first creation for The Sims community, so I don't get any recognition, which in turn doesn't generate me any financial return at all. If in the future Dream Launcher becomes more recognized and starts to give a return, even if minimal, in donations, I might make an exception and release a port or something similar.

In short, this is basically a project of love, and if I decided to expand it to other versions, I wouldn't be able to maintain it, so I prefer to only keep active what I can really take care of! 🙂

PS: Dream Launcher is also open source, so if you know anyone who wants to fork or port it, everyone is free to do so!

**Comment by justcallmedots** [comment]
Who said anything about pirate! I pay for EA play and also I buy some of the games via allkeyshop and I buy them for like 5-10 euros a pop like one every 3 months. If Skyrim modders can support Skyrim Special Edition as well as mods for Skyrim's AE/anniversary edition at the same time, why can't you? Your profile pic says you are young. You have a lifetime ahead of you. Please help this "old fart" before he "kicks the bucket". There is and always will be a debate/war on piracy whenever it is steam the most up to date version or ea app/origin version is latest one. That will never change. The game will never update or get new official content. I was hoping that you were more open minded person and see that the world is not just now, just the present, but also for the loong run. If afraid of piracy then you can make it so people need to choose platform and login in order to use the app.

**Comment by marcos4503** [comment]
I understand! Sorry for the misunderstanding. It's just that several people from different countries have already asked me (sometimes in private, sometimes on other social networks) about possible support for the Pirated version or even the DVD version, and so, I wanted to cover the answer for other editions as well, for possible people who might see this topic in the future. I should have mentioned this in my first post, sorry!

I have no problem with people asking, but I wanted to cover the answer anyway.

But, regarding support, precisely because I'm looking to the future, I can't cover support for other editions of The Sims 3, because I won't be able to maintain support in an efficient and healthy way. Look, The Sims 3 on Steam is on patch 1.67, while The Sims 3 from EA/Origin is on patch 1.69. It may seem like a small difference, but version 1.69 has many changes in points of the game. An example of this is the fact that the Steam version doesn't have DRM, while the EA/Origin version has DRM. These differences are reflected in things like core mods having to have versions for 1.67 and 1.69, different ways of applying bug fixes or patches, etc.

Because of these differences, if I were to support the EA/Origin version, it would be as if I had to release a "Dream Launcher Origin Edition", because I would need to rewrite several and several things in the Launcher, adapt bugfixes, mods, patches, etc. It would be a very big job, and of course, all of this would be accompanied by perhaps hundreds of additional hours of testing and more testing, because software must be tested in the battlefield to ensure its efficiency.

I understand that many modders offer support for all editions of the game, etc. But it is necessary to keep in mind that most games, even if they are updated to new editions, or sold in different stores, have the same core. Using as an example, Need For Speed ​​Heat, also from EA, it is sold on Steam, Epic and EA/Origin. It is very likely that it will be the same, even if downloaded from different stores, however, this rule does not apply to The Sims 3, unfortunately.

But in the end, every modder supports their creations according to what they can afford. Some people have more free time than others, some modders have more incentive to do so, etc. In my case, I have very little free time, the only version I play and know a LOT about is the Steam version. Because of my limited free time, I can only support the version that I play, so it becomes easier to release updates, patches, do testing, and so on. Adding to that is the fact that Dream Launcher is small, so there isn't a big enough demand to motivate me to make a new version of the Launcher.

I hope I've managed to summarize and explain the nuances well here. Anyway, these are some of the reasons why I can't offer support for any other (or all) editions of The Sims 3. So I'm really sorry, but for me, at the moment, it's impossible to extend this support to other versions of the game. 😕
